### PR TITLE
CLOUDP-155416: Add flag to update cluster termination protection

### PR DESCRIPTION
**Pull request by AndreiMatei2206**
## Proposed changes

Add flags to enable/disable cluster termination protection with `cluster update` command. The enable and disable flags are mutually exclusive.
Screenshot 2023-01-27 at 13 55 04

_Jira ticket:_ CLOUDP-155416

## Checklist

- [x] I have signed the [MongoDB CLA](https://www.mongodb.com/legal/contributor-agreement)
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added any necessary documentation in document requirements section listed in [CONTRIBUTING.md](https://github.com/mongodb/mongodb-atlas-cli/blob/master/CONTRIBUTING.md) (if appropriate)
- [ ] I have addressed the @mongodb/docs-cloud-team comments (if appropriate)
- [ ] I have updated [test/README.md](https://github.com/mongodb/mongodb-atlas-cli/blob/master/test/README.md) (if an e2e test has been added)
- [x] I have run `make fmt` and formatted my code

### CODE REVIEW

File: internal/cli/atlas/clusters/update.go [hunk 1]
```diff
@@ -167,6 +170,10 @@ You can't change the name of the cluster or downgrade the MongoDB version of you
 	cmd.Flags().StringVar(&opts.mdbVersion, flag.MDBVersion, "", usage.MDBVersion)
 	cmd.Flags().StringVarP(&opts.filename, flag.File, flag.FileShort, "", usage.ClusterFilename)
 
+	cmd.Flags().BoolVar(&opts.enableTerminationProtection, flag.EnableTerminationProtection, false, usage.EnableTerminationProtection)
+	cmd.Flags().BoolVar(&opts.disableTerminationProtection, flag.DisableTerminationProtection, false, usage.EnableTerminationProtection)
```
Comment (sarahsimpers) [inline internal/cli/atlas/clusters/update.go:174]:
```suggestion
	cmd.Flags().BoolVar(&opts.disableTerminationProtection, flag.DisableTerminationProtection, false, usage.DisableTerminationProtection)
```

**Review by sarahsimpers — CHANGES_REQUESTED** [review]
Copy looks good but wrong flag text used for the docs.

File: internal/cli/atlas/clusters/update.go (hunk 1 above)
Comment (AndreiMatei2206) [reply internal/cli/atlas/clusters/update.go:174]: Thank you for spotting that 😅

**Review by gssbzn — APPROVED** [review]
LGTM

**Review by sarahsimpers — APPROVED** [review]

**Pull request merged**